docs/management-ui.mdx

```diff
@@ -65,3 +65,35 @@ curl -X POST "https://api.svix.com/api/v1/auth/dashboard_access/app_Xzx8bQeOB1D1
 
 </TabItem>
 </CodeTabs>
+
+
+## Embedding in your own dashboard
+
+The returned URL from the previous section can also be embedded in your own dashboard using an iframe. This is how it looks like:
+
+<img alt="Embedded Management UI screenshot" src={useBaseUrl("img/embedded-management-ui.png")} />
+
+To do it, just set the url from the examples above in the `src` property of the iframe:
+
+```html
+<iframe
```

Comment:
I would remove this one and just include the example with the basic styling.

Reply:
Done.